expression/testdata/expression_suite_out.json

```diff
@@ -274,6 +274,18 @@
           "    └─TableReader_11(Probe) 10000.00 root  data:TableFullScan_10",
           "      └─TableFullScan_10 10000.00 cop[tikv] table:t1 keep order:false, stats:pseudo"
         ]
+      },
+      {
+        "SQL": "explain select * from t1 left join t2 on t1.a = t2.a where ifnull(t2.b, t1.a) = 1",
+        "Result": [
+          "Selection_7 9990.00 root  eq(ifnull(test.t2.b, test.t1.a), 1)",
```

Comment:
[optional] Can we do twice selections? The first is pushed down, the second as the root for this one?